dracut/parse-kickstart

```diff
@@ -553,6 +553,10 @@ def ksnet_to_ifcfg(net, filename=None):
             ifcfg['BRIDGING_OPTS'] = " ".join(keyvalues)
 
         for i, slave in enumerate(net.bridgeslaves.split(","), 1):
+            # Handle virtual slaves created by preceding network commands (eg bond)
+            if update_virtual_slave_ifcfg(slave,
+                    setting="BRIDGE=%s" % dev):
```

Comment:
Could you please remove the newline here, I think it will be more readable without it .